### DISCUSSION TITLE
Absence of correct version in `Literal support`

### DISCUSSION THREAD

**Discussion by akonitx** [sqlalchemy/sqlalchemy#9890] [Usage Questions]
Is that normal that latest version on Jun 3 2023, in documentation about Declarative Table, in the section
[Using Python Enum or pep-586 Literal types in the type map](https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#using-python-enum-or-pep-586-literal-types-in-the-type-map), in the line with **_New in version 2.0.1: - Added Literal support_** has been written version `2.0.1` which does not exist at that time, because latest release is `2.0.15`?

**Comment by zzzeek** [comment]
the documentation note means that support for `typing.Literal` will work as of SQLAlchemy version 2.0.1 or greater.    So, 2.0.1, 2.0.2, 2.0.3, 2.0.4, etc. up through 2.0.15.  If you have 2.0.0, wont work.

is that what you're asking?

**Reply by akonitx** [reply]
oh, sorry. Yes it is. Sometimes i'm blind.
